core/src/test/resources/org/bitcoinj/script/script_invalid.json

```diff
@@ -151,8 +151,8 @@
 ["2 2 LSHIFT", "8 EQUAL", "P2SH,STRICTENC", "disabled"],
 ["2 1 RSHIFT", "1 EQUAL", "P2SH,STRICTENC", "disabled"],
 
-["1","NOP1 NOP2 NOP3 NOP4 NOP5 NOP6 NOP7 NOP8 NOP9 NOP10 2 EQUAL", "P2SH,STRICTENC"],
```

Comment:
The original can be found at https://github.com/bitcoin/bitcoin/blob/master/src/test/data/script_tests.json. The file has been merged from script_valid.json and script_invalid.json in with https://github.com/bitcoin/bitcoin/commit/dde46d3ae1de18700e089bc1861cf1e18649dc5d#diff-a79f02a42f31e934e3bdb4f40500b908, something we should follow too (maybe even before merging this PR). Anyway, I suggest gathering as much CSV-related test data from the original file as possible, because your code needs to pass these tests.